presidio_evaluator/data_generator/presidio_data_generator.py

```diff
@@ -122,7 +125,7 @@ def parse(
     def read_template_file(templates_file):
         with open(templates_file) as f:
             lines = f.readlines()
-            lines = [line.replace("\\n", "\n") for line in lines]
+            lines = [line.strip() for line in lines]
```

Comment:
No, The templates currently in there don't have `\\n` in them
I'm not sure what scenario this is accounting for? Is there a known dataset that this needs done for?
`\\n` would imply the writer is intentionally escaping the escape character to avoid the new line from existing

Reply:
Not sure, but I would keep the line just for the continuity.

Reply:
I've added `strip()` which removes new-line characters at the start/end of the line since these are artefacts of being a file
If I retain `replace("\\n", "\n")` then if `\\n` appears at the start of end of the line it will now be removed
This feels strange to me, as the extra escape character suggests this is there intentionally as part of the template and not an artefact of the encoding
Happy to re-include it if you're happy that `\\n` strings at the start/end of of lines will be removed